test.mk

```diff
@@ -161,50 +162,256 @@ spec:
           serviceName: {{NAME}}
           servicePort: 80
 endef
+define TEST_SERVICE_SPEC_INGRESS_CLOUDEP
+$(call TEST_INGRESS)
+--- 
+$(call TEST_SERVICE)
+  targetIngress:
+    name: {{INGRESS_NAME}}
+    namespace: default
+    jwtServices:
+    - {{NAME}}
+---
+$(call TEST_APP)
+endef
+define TEST_SERVICE_SPEC_CONFIGMAP_SPEC
+apiVersion: v1
+kind: ConfigMap 
+metadata: 
+  name: swagger 
+data: 
+  spec: |
```

Comment:
Use the block string `|-` instead of `|` to remove the line feed, otherwise, the first line of the generated spec is blank. 

I think consumers of the spec like ESP can tolerate this but it's better to keep it clean if possible.